### DISCUSSION TITLE
Order option into mission file

### DISCUSSION THREAD

**Discussion by nivcoo** [BG-Software-LLC/SuperiorSkyblock2#733] [Implemented Feature Requests] [answered]
I can't reply to my previous discussion https://pics.revolis.fr/2021/11/14/11-43_dbIf.png so...
If i do this request it's because if i change the name all my players progression will be reset. 

Hello, before we have all mission in same file so we could ordered the gui like we want but now it's messy because it's based on the name order, can you add order: number option to place the missions like we want ? Thx

**Comment by OmerBenGera** [comment, marked as answer]
Add "weight: X" to your mission files.
Missions will be ordered by their weights. Two missions with the same weight will be ordered by their names instead.
All missions have a default weight of 0.
